chance/chance.d.ts

```diff
@@ -115,7 +116,8 @@ declare namespace Chance {
         mixin(desc: MixinDescriptor): any;
         pad(num: number, width: number, padChar?: string): string;
         pick<T>(arr: T[]): T;
-        pick<T>(arr: T[], count: number): T[];
+        pick<T>(arr: T[], count: number): T | T[];
```

Comment:
This seems to be accurate based on the source - but been deprecated in favor of pickone() and pickset().  @madslunt can you include pickone() as well?